static/searchPage.html

```diff
@@ -3,8 +3,9 @@
   <head>
     <meta charset="UTF-8" />
     <title>Ogene - Group Search</title>
-    <link rel="stylesheet" href="/css/searchPage.css" />
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDDhfcuk15Apx3i72mFSilulsPtJReGhcY"></script>
```

Comment:
This tutorial recommends using async and defer with the script - though this may complicate later use. Is this an option you want to investigate?

https://developers.google.com/maps/documentation/javascript/tutorial

Reply:
The maps JavaScript API had a `callback` parameter used to call a function when the API loads completely. From my point of understanding `async defer` may not be necessary because we are't using the callback parameter. The map and result will be loaded only when the user inputs their search information.
Its purpose there is to render the html page while waiting for the API to load the callback parameter.

Reply:
If it ends up being important, we could always switch it up. I was also considering making the API call dynamically from the js file.

Reply:
Sounds good, this can always be revisited later.